Login Message Improvement and improvment of login response

estevan83 opened the pull request:
this commit allow you to have a real error message instead null message after login errors.

https://i.imgur.com/nQN8Ldg.png

After login I'm able to know (for further query) the id of the logged user
https://i.imgur.com/pRYTqFh.png

Commit:
1)Improve login messagge info
2)After login retrieve user id instead only token

Commit:
1)Improve login messagge info
2)After login retrieve user id instead only token

Commit:
Retrieve list of Users
Retrieve single User by id

Commit:
Revert "Retrieve list of Users Retrieve single User by id"

This reverts commit 4078727a30040c0a4fa031ad677daf4ac3b94837.

Commit:
Retrieve list of Users
Retrieve single User by id

sardoj commented:
Thank you so mush for you improvement.
Please, can you had a unit test or a script to test your update?

Commit:
Retrieve related documents and comment during call to retrieve single detail view

Commit:
Add support for related activity

Commit:
Added function to upload document

Commit:
Added function to retrieve document content

Commit:
Added function to retrieve document content (url and file content instead document content)

Commit:
MODTRACKER SUPPORT
Added support to retrieve full list of modification (like history in crm)
Added support to retrieve Modtracker list of single Entity

Commit:
Adjust like behaviour in Like query

Commit:
Add support for VCQ (Vtiger custom query)
Allow to obtain data from custom query
